spec/models/product_share_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe ProductShare, type: :model do
+  subject (:product_share) { create(:product_share) }
+
+  it 'is valid with valid attributes' do
+    expect(build(:product_share)).to be_valid
+  end
+
+  it 'is not valid without a platform' do
+    expect(build(:product_share, platform: nil)).not_to be_valid
+  end
+
+  it 'is not valid without a product id' do
+    expect(build(:product_share, product_id: nil)).not_to be_valid
+  end
```

Comment:
Para este y el de abajo es más fácil testear directamente las validaciones: 

`it { is_expected }`

```
describe 'validations' do
    it { is_expected.to validate_presence_of(:platform) }
    it { is_expected.to belong_to(:description) }
  end
```

De eso último no estoy seguro de la sintaxis

Reply:
Que buen tip! Es mucho mas directo así, no se me había ocurrido (estoy recien iniciandome con los tests en rspec)!. Gracias!